bird-slideshow.py

```diff
@@ -17,7 +17,7 @@
 from bs4 import BeautifulSoup
 
 class Config:
-    def __init__(self, config_file: str | None) -> None:
+    def __init__(self, config_file=None):
```

Comment:
This looks good.  Thanks.

Reply:
Vertical Bar was introduced for Union types in 3.10